packages/odyssey-react-mui/src/theme/theme.ts

```diff
@@ -24,13 +23,18 @@ import "./components.types";
 import "./mixins.types";
 import "./palette.types";
 import "./typography.types";
+import { TokenOverrideOptions } from ".";
 
-export const theme = createTheme({
-  components,
-  mixins,
-  palette,
-  shape,
-  spacing,
-  temp,
-  typography,
-}) as OdysseyTheme;
+export type { OdysseyTheme } from "./OdysseyTheme";
+
+export const createOdysseyTheme = (tokenOverrides?: TokenOverrideOptions) => {
+  return createTheme({
+    components,
+    mixins: mixins(tokenOverrides),
+    palette: palette(tokenOverrides),
+    shape: shape(tokenOverrides),
+    spacing: spacing(tokenOverrides),
+    temp,
+    typography: typography(tokenOverrides),
+  });
+};
```

Comment:
I would have this function take tokens, not the overrides. We can merge tokens with the overrides in the Provider component.